data/interfaces/default/home_stats.html

```diff
@@ -133,7 +135,7 @@ <h4>${top_stat['stat_title']}</h4>
                             % for row in top_stat['rows']:
                             <li class="dashboard-stats-info-item ${'expanded' if loop.index == 0 else ''}" data-stat_id="${stat_id}"
                                 data-rating_key="${row.get('rating_key')}" data-guid="${row.get('guid')}" data-title="${row.get('title')}"
-                                data-art="${row.get('art')}" data-thumb="${row.get('thumb')}" data-platform="${row.get('platform_name')}"
+                                data-art="${row.get('art')}" data-thumb="${row.get('thumb')}" data-platform="${row.get('platform_name')}" data-library-type="${row.get('type')}"
```

Comment:
Shall it stay library-type or be re-named to section-type? I would leave it like that.

Reply:
This is fine.